commit 8d7b3ae5db94d8dde759892a8a17d5c367b826b9 (refs/changes/86/286/3)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-08-12T21:26:27+00:00 (3 years, 2 months ago)
    
    DOCKER-774 docker build omits all but one hard linked file from images
    DOCKER-751 docker build failure for files with adjusted hard links
    DOCKER-808 docker commit cannot create images that include a socket
    DOCKER-902 zfs_snapshot_tar keeps mum on unexpected zfs diff output
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/deps/zfs_snapshot_tar b/deps/zfs_snapshot_tar
index ae9c82c..7603eca 160000
--- a/deps/zfs_snapshot_tar
+++ b/deps/zfs_snapshot_tar
@@ -1 +1 @@
-Subproject commit ae9c82cfa0109b27e5286196d3a5281d3a2b86ec
+Subproject commit 7603eca7c3475598d4f543ac61cd0e9d7dfa7d9b
diff --git a/package.json b/package.json
index e4566f2..337d85e 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "cn-agent",
-  "description": "SmartDataCenter Compute Node Agent",
-  "version": "1.5.2",
+  "description": "Triton Compute Node Agent",
+  "version": "1.5.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
